pkg/ovs/openflow/interfaces.go

```diff
@@ -65,23 +63,20 @@ const (
 // Bridge defines operations on an openflow bridge.
 type Bridge interface {
 	CreateTable(id, next TableIDType, missAction MissActionType) Table
-	GetName() string
 	DeleteTable(id TableIDType) bool
 	DumpTableStatus() []TableStatus
-	// Connect initiates connection to the OFSwitch. It will block until the connection is established.
-	// If Bridge is not connected in maxRetry times, it will return error.
-	Connect(maxRetry int) error
+	// DumpFlows queries the Openflow entries from OFSwitch. The filter of the query is Openflow cookieID; the result is
+	// a map from flow cookieID to FlowStates.
+	DumpFlows(cookieID, cookieMask uint64) map[uint64]*FlowStats
```

Comment:
which one is correct? FlowStates in comment or FlowStats in code?

Reply:
I will change to FlowStates in code.